dev/MRTCore/mrt/mrm/include/mrm/build/FileBuilder.h

```diff
@@ -120,7 +120,7 @@ class FileBuilder : public DefObject
     UINT32 GetDataSize() { return m_cbData; }
 
 public:
-    static HRESULT CreateInstance(DEFFILE_MAGIC magic, BaseFile::SectionCount sizeSections, _Outptr_ FileBuilder** result);
+    static HRESULT CreateInstance(__in DEFFILE_MAGIC magic, __in BaseFile::SectionCount sizeSections, _Outptr_ FileBuilder** result);
```

Comment:
>__in [](http://example.com/codeflow?start=34&length=4)

in should not be necessary

Reply:
In FileBuilder.cpp, it defines

HRESULT FileBuilder::CreateInstance(__in DEFFILE_MAGIC magic, __in BaseFile::SectionCount sizeSections, _Outptr_ FileBuilder** result)